File: dsa-practice/maths/problem2.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
    cin>>n;
    int result=0;
    bool resultfound = false;
    int i=0;
    while(resultfound==false){
        if(i*i==n){
            result =i;
            resultfound=true;
        }
        i++;
    }
    cout<<"Square root of the number is: "<<result<<endl;
    return 0;
}

File: dsa-practice/maths/problem2.js
// Write a program to find the square root of a number.
const sqrt = (num)=>{
    let res;
    let i = 0;
    let found = true;
    while(found){
        if(i*i===num){
            res = i
            found = false
        }
        i++;
    }
    return res
}
console.log(sqrt(4))